DAO.sol

```diff
@@ -496,7 +508,7 @@ contract DAO is DAOInterface, Token, TokenSale {
         }
 
         if (p.newCurators) {
-            if (now > p.votingDeadline + 28) {
```

Comment:
@CJentzsch I introduce a logic change here, but I think this was a typo so it's safe.